app/app/models/term.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import Boolean, Column, Date, ForeignKey, Integer, String
+
+from app.db.base_class import Base
+from app.models.year import Year
+
+
+class Term(Base):
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(100), index=True)
```

Comment:
Shouldn't this be set `nullable=False`? Considering it's required in TermCreate schema.